### PR TITLE
data: malicodes2

### PR DESCRIPTION
**Pull request by malicodes2**

### CODE REVIEW

File: data/malicodes2.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/malicodes",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Blogs",
+      "url": "https://malicodes.hashnode.dev/",
+      "icon": "SiHashnode"
+    }
+  ]
```
Comment (Aadarsh805) [inline data/malicodes2.json:34]: Hey you're missing a `}` at the end

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/malicodes2.json (hunk 1 above)
Comment (malicodes2) [reply data/malicodes2.json:34]: Adding...!

Comment (malicodes2) [reply data/malicodes2.json:35]: Done

Comment (dan-mba) [reply data/malicodes2.json:35]: The `}` on line 34 needs to be moved after the `]` on line 35

Comment (malicodes2) [reply data/malicodes2.json:35]: Oh I apologize. Thanks for pointing out.

**Comment by malicodes2** [comment]
Done

**Review by dan-mba — APPROVED** [review]
Looks good to me

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by eddiejaoude** [comment]
Looks good 👍 

Great collaboration 💪 

You can also join us in the EddieHub Discord http://discord.eddiehub.org

**Pull request merged**